### PR TITLE
Add directory API

### PR DESCRIPTION
**Pull request by bryanjonker-illinois**

### CODE REVIEW

File: ContactAppCore/ViewModel/EmployeeActivityInformation.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using ContactAppCore.Data.Models;
+
+namespace ContactAppCore.ViewModel
+{
+    public class EmployeeActivityInformation
+    {
+        public EmployeeActivityInformation()
+        {
```
Comment (aobanion) [inline ContactAppCore/ViewModel/EmployeeActivityInformation.cs:8]: Not needed

File: ContactAppCore/ViewModel/EmployeeInformation.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ContactAppCore.ViewModel
+{
+    public class EmployeeInformation
+    {
+        public EmployeeInformation()
+        {
+        }
```
Comment (aobanion) [inline ContactAppCore/ViewModel/EmployeeInformation.cs:11]: not need

File: ContactAppCore/ViewModel/EmployeeInformation.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+        public EmployeeInformation(JobProfile profile)
+        {
+            Biography = string.IsNullOrWhiteSpace(profile.Biography) ? profile.EmployeeProfile.Biography ?? "" : profile.Biography;
+            CVUrl = profile.EmployeeProfile.CVUrl ?? "";
```
Comment (aobanion) [inline ContactAppCore/ViewModel/EmployeeInformation.cs:16]: string.empty versus ""

File: ContactAppCore/ViewModel/EmployeeActivityInformation.cs (hunk 1 above)
Comment (bryanjonker-illinois) [reply ContactAppCore/ViewModel/EmployeeActivityInformation.cs]: Changed!

File: ContactAppCore/ViewModel/EmployeeInformation.cs (hunk 2 above)
Comment (bryanjonker-illinois) [reply ContactAppCore/ViewModel/EmployeeInformation.cs]: Changed!

File: ContactAppCore/ViewModel/EmployeeInformation.cs (hunk 3 above)
Comment (bryanjonker-illinois) [reply ContactAppCore/ViewModel/EmployeeInformation.cs]: Changed!

File: ContactAppCore/ViewModel/EmployeeInformation.cs [hunk 4]
```diff
@@ -6,21 +6,17 @@ namespace ContactAppCore.ViewModel
...
             Biography = string.IsNullOrWhiteSpace(profile.Biography) ? profile.EmployeeProfile.Biography ?? "" : profile.Biography;
-            CVUrl = profile.EmployeeProfile.CVUrl ?? "";
+            CVUrl = profile.EmployeeProfile.CVUrl ?? string.Empty;
             InternalOrder = profile.InternalOrder;
-            NetId = profile.EmployeeProfile.Title ?? "";
+            NetId = profile.EmployeeProfile.Title ?? string.Empty;
             OfficeName = profile.Office.Title;
-            OfficeCode = profile.Office.InternalCode ?? "";
+            OfficeCode = profile.Office.InternalCode ?? string.Empty;
             Phone = profile.Phone ?? "";
```
Comment (aobanion) [inline ContactAppCore/ViewModel/EmployeeInformation.cs:17]: miss one.

**Review by aobanion — APPROVED** [review]

**Review by merrifld — APPROVED** [review]

**Pull request merged**